template/src/pages/video-call/VideoComponent.tsx

```diff
@@ -40,7 +44,22 @@ const VideoComponent = () => {
     isValidReactComponent(layoutsData[layout].component)
   ) {
     const CurrentLayout = layoutsData[layout].component;
-    return <CurrentLayout renderData={activeUids} />;
+    return (
+      <View
```

Comment:
@SupriyaAdep this view and style are not needed if more than 1 user is in the call right?  in that case can we just return <CurrentLayout renderData={activeUids} />; so we can avoid extra view and styling